docs/administration/configuring-replication/create-replication-rules.md

```diff
@@ -8,15 +8,15 @@ A replication endpoint must exist before you create a replication rule. To creat
 1. Log in to the Harbor interface with an account that has Harbor system administrator privileges.
 1. Expand **Administration**, and select **Replications**.
 
-   ![Add a replication rule](../../../img/replication-rule1.png)
```

Comment:
../../../ should be correct, it should be one more level than the actual folder structure, the image has its own directory basically, please double check